avm/res/network/express-route-circuit/main.bicep

```diff
@@ -245,6 +253,16 @@ output name string = expressRouteCircuit.name
 @description('The service key of the express route circuit.')
 output serviceKey string = expressRouteCircuit.properties.serviceKey
 
+@description('The service provider provisioning state of the express route circuit.')
+output serviceProviderProvisioningState string = expressRouteCircuit.properties.serviceProviderProvisioningState
+
+@description('The authorizations for the express route circuit.')
+output authorizations array = map(expressRouteCircuit.properties.authorizations, (authorization, id) => {
+  id: authorization.id
+  name: authorization.name
+  key: authorization.properties.authorizationKey
```

Comment:
Would the authorization key not be a sensitive bit of information? If so, mind you, that Bicep (as of today) does not support `secure()` outputs

Reply:
I don't believe that it is considered to be sensitive as they are shown in the portal as plaintext.

Reply:
Well... storage access keys are too 😄 The portal is unfortuantely not a reliable source.

@arnoldna, what's your take on this? If we end up agreeing it **is** in fact sensitive, we only have 2 choices:
1. Remove it from the output and implement AVM's KeyVault export feature to store this item in a key vault during dpeloyment - OR
2. Remove it from the output and ask users to fetch the key using e.g. an `existing` resource reference to fetch the key on the fly (or maybe there even is a function, like `listKeys()`)

Reply:
Sorry what I meant was there's no visual obfuscation with the authorisation keys (I.e. a show/hide button). Like with storage access keys, but I get your point about not replying on the portal.

Yeah, if it is considered to be sensitive. I think the second option is better and I can picture how doing an existing resource reference for `microsoft.network/expressroutecircuits/authorisations` would work.

Reply:
I am inclined to believe that it is sensitive information, and we shouldn't be outputting it. I am in favor of the second option as well.

Reply:
@arnoldna thanks for the confirmation. Will remove authorisations from outputs.